simulators/ethereum/engine/enginetests.go

```diff
@@ -226,108 +227,103 @@ var engineTests = []TestSpec{
 		SlotsToFinalized: big.NewInt(20),
-		Run:              invalidMissingAncestorReOrgGen(9, InvalidStateRoot, false, true),
+		Run:              invalidMissingAncestorReOrgGen(9, InvalidStateRoot, true),
 	},
 	{
 		Name:             "Invalid Ancestor Chain Re-Org, Invalid StateRoot, Invalid P10', Reveal using newPayload",
 		SlotsToFinalized: big.NewInt(20),
-		Run:              invalidMissingAncestorReOrgGen(10, InvalidStateRoot, false, true),
+		Run:              invalidMissingAncestorReOrgGen(10, InvalidStateRoot, true),
 	},
 
 	// Invalid Ancestor Re-Org Tests (Reveal via sync through secondary client)
 	{
 		Name:             "Invalid Ancestor Chain Re-Org, Invalid StateRoot, Invalid P9', Reveal using sync",
-		TimeoutSeconds:   15,
 		SlotsToFinalized: big.NewInt(20),
-		Run:              invalidMissingAncestorReOrgGen(9, InvalidStateRoot, true, false),
+		TimeoutSeconds:   15,
+		Run:              invalidMissingAncestorReOrgGenSync(9, InvalidStateRoot, false),
 	},
 	{
 		Name:             "Invalid Ancestor Chain Re-Org, Invalid StateRoot, Empty Txs, Invalid P9', Reveal using sync",
-		TimeoutSeconds:   15,
 		SlotsToFinalized: big.NewInt(20),
-		Run:              invalidMissingAncestorReOrgGen(9, InvalidStateRoot, true, true),
+		TimeoutSeconds:   15,
+		Run:              invalidMissingAncestorReOrgGenSync(9, InvalidStateRoot, true),
 	},
 	{
-		Name:             "Invalid Ancestor Chain Re-Org, Invalid ReceiptsRoot, Invalid P9', Reveal using sync",
-		TimeoutSeconds:   15,
+		Name:             "Invalid Ancestor Chain Re-Org, Invalid ReceiptsRoot, Invalid P8', Reveal using sync",
 		SlotsToFinalized: big.NewInt(20),
-		Run:              invalidMissingAncestorReOrgGen(9, InvalidReceiptsRoot, true, false),
+		TimeoutSeconds:   15,
+		Run:              invalidMissingAncestorReOrgGenSync(8, InvalidReceiptsRoot, false),
 	},
 	{
 		Name:             "Invalid Ancestor Chain Re-Org, Invalid Number, Invalid P9', Reveal using sync",
-		TimeoutSeconds:   15,
 		SlotsToFinalized: big.NewInt(20),
-		Run:              invalidMissingAncestorReOrgGen(9, InvalidNumber, true, false),
+		TimeoutSeconds:   15,
+		Run:              invalidMissingAncestorReOrgGenSync(9, InvalidNumber, false),
 	},
 	{
-		Name:             "Invalid Ancestor Chain Re-Org, Invalid GasLimit, Invalid P9', Reveal using sync",
-		TimeoutSeconds:   15,
+		Name:             "Invalid Ancestor Chain Re-Org, Invalid GasLimit, Invalid P8', Reveal using sync",
 		SlotsToFinalized: big.NewInt(20),
-		Run:              invalidMissingAncestorReOrgGen(9, InvalidGasLimit, true, false),
+		TimeoutSeconds:   15,
+		Run:              invalidMissingAncestorReOrgGenSync(8, InvalidGasLimit, false),
 	},
 	{
-		Name:             "Invalid Ancestor Chain Re-Org, Invalid GasUsed, Invalid P9', Reveal using sync",
-		TimeoutSeconds:   15,
+		Name:             "Invalid Ancestor Chain Re-Org, Invalid GasUsed, Invalid P8', Reveal using sync",
```

Comment:
Why are some `P8'` instead of `P9'`?

Reply:
Because I am unable to mine invalid blocks on top of an invalid block on a node. The next block is send via NewPayload. See also description